Armadillo/src/main/java/com/scribd/armadillo/download/DownloadEngine.kt

```diff
@@ -70,8 +70,7 @@ internal class ExoplayerDownloadEngine @Inject constructor(private val context:
         })
     }
 
-    override fun removeDownload(audiobook: AudioPlayable) =
-        DownloadService.sendRemoveDownload(context, downloadService, audiobook.request.url, true)
+    override fun removeDownload(audiobook: AudioPlayable) = downloadManager.removeDownload(audiobook.request.url)
```

Comment:
I think `removeDownload` can be nontrivial in runtime. Should we perhaps make this launch a job to do it?